.circleci/config.yml

```diff
@@ -0,0 +1,98 @@
+version: 2.1
+executors:
+  default:
+    working_directory: ~/go/src/github.com/mattermost/mmetl
+    docker:
+      - image: circleci/golang:1.13
+
+jobs:
+  lint:
+    executor:
+      name: default
+    steps:
+      - checkout
+      - run:
+          command: |
+            echo "Installing mattermost-govet"
+            GO111MODULE=off go get -u github.com/mattermost/mattermost-govet
```

Comment:
Maybe this is a good time to introduce golangci-lint to this repo similar how it was done on `mmctl`: https://github.com/mattermost/mmctl/pull/107